src/ceph.in

```diff
@@ -913,17 +921,14 @@ def main():
         print(str(e), file=sys.stderr)
         return 1
 
-    if parsed_args.help:
```

Comment:
yes, we print `Monitor commands: ===== [Contac..........]` here, without any command explain.  the problem is that:

1. we print header of help command at first
2. we check `ping` command. ( `ceph ping --help` exits here)
2. we print boby of help command.

we should put  1 and 3 together.